DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using LT.DigitalOffice.CompanyService.Mappers.Interfaces;
+using LT.DigitalOffice.CompanyService.Models;
+using LT.DigitalOffice.CompanyService.Repositories;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using LT.DigitalOffice.CompanyServiceUnitTests.UnitTestLibrary;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Repositories
+{
+    class CompanyRepositoryTests
+    {
+        private CompanyServiceDbContext dbContext;
+        private ICompanyRepository repository;
+        private Mock<IMapper<DbPosition, Position>> mapperMock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var dbOptions = new DbContextOptionsBuilder<CompanyServiceDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new CompanyServiceDbContext(dbOptions);
+            mapperMock = new Mock<IMapper<DbPosition, Position>>();
+            repository = new CompanyRepository(dbContext, mapperMock.Object);
+
+            var dbPosition = new DbPosition
+            {
+                Id = Guid.NewGuid(),
+                Name = "Position",
+                Description = "Description"
+            };
+            dbContext.Positions.Add(dbPosition);
+            dbContext.SaveChanges();
+            mapperMock.Setup(mapper => mapper.Map(dbPosition))
+                .Returns(new Position
+                {
+                    Name = dbPosition.Name,
+                    Description = dbPosition.Description
+                });
+        }
+
+        [TearDown]
+        public void CleanDb()
+        {
+            if (dbContext.Database.IsInMemory())
+            {
+                dbContext.Database.EnsureDeleted();
+            }
+        }
+
```

Comment:
Add a test that checks to throw an exception if the position is not in the database